src/test-e2e/application/application-delete-e2e.spec.ts

```diff
@@ -78,8 +78,8 @@ describe('Application Delete', function () {
       // We created the app after the wall loaded, so refresh to make sure app wall shows the new app
       appWall.appList.refresh();
 
-      let appCount = 0;
-      appWall.appList.getTotalResults().then(count => appCount = count);
+      appWall.appList.header.setSearchText(testAppName);
+      expect(appWall.appList.getTotalResults()).toBe(1);
 
       e2e.sleep(5000);
```

Comment:
I know these aren't due to your PR - but why do we have two 5s delays ? We should avoid using sleep... we must be waiting for something, so we should wait for it.